engine/dlib/src/dlib/transform.h

```diff
@@ -29,13 +27,13 @@ namespace dmTransform
     class TransformS1
     {
         // tx, ty, tz, scale
-        Vector4 m_TranslationScale;
-        Quat    m_Rotation;
+        dmVMath::Vector4 m_TranslationScale;
+        dmVMath::Quat    m_Rotation;
```

Comment:
We never want to "open up" namespaces in a header file, even in another namespace.
If makes the "ownership" unclear, and is may hinder refactoring etc.

I.e. `dmTransform::Vector4` shouldn't be allowed. Neither should "Vector4" (if it was added to the global scope)